cloudera/datadog_checks/cloudera/config.py

```diff
@@ -3,18 +3,19 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 
 # Discovery class requires 'include' to be a dict, so this function is needed to normalize the config
-def normalize_config_clusters_include(log, clusters_config):
-    config = {}
-    if clusters_config and clusters_config.include:
-        if not isinstance(clusters_config.include, list):
-            raise TypeError('Setting `include` must be an array')
-        for entry in clusters_config.include:
-            log.debug("entry: %s", type(entry))
-            if isinstance(entry, str):
-                config[entry] = None
-            elif isinstance(entry, dict):
-                for key, value in entry.items():
-                    config[key] = value.copy()
-            else:
-                raise TypeError('`include` entries must be a map or a string')
-    return config
+def normalize_discover_config_include(log, config):
+    normalized_config = {}
+    log.debug("normalize_discover_config_include: %s", type(config))
+    include_list = config.get('include') if isinstance(config, dict) else config.include if config else []
+    log.debug("normalize_discover_config_include: %s", include_list)
```

Comment:
Should these log lines be different since both are `normalize_discover_config_include` but the values are different.